integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/http/inbound/transport/test/HttpInboundDispatchTestCase.java

```diff
@@ -37,175 +30,64 @@
  */
 
 public class HttpInboundDispatchTestCase extends ESBIntegrationTest {
-    private LogViewerClient logViewerClient = null;
+
+    private static final String requestPayload = "<soapenv:Envelope xmlns:soapenv='http://schemas.xmlsoap.org/soap/envelope/' >"
```

Comment:
```suggestion
private static final String REQUEST_PAYLOAD = "<soapenv:Envelope xmlns:soapenv='http://schemas.xmlsoap.org/soap/envelope/' >"
```

Reply:
Fixed in https://github.com/wso2/micro-integrator/pull/580